pkg/protocol/engine/mempool/spenddag/tests/assertions.go

```diff
@@ -0,0 +1,87 @@
+package tests
+
+import (
+	"github.com/stretchr/testify/require"
+)
+
+// Assertions provides a set of assertions for the ConflictDAG.
+type Assertions struct {
+	f *Framework
+}
+
+// Children asserts that the given spend has the given children.
+func (a *Assertions) Children(spendAlias string, childAliases ...string) {
+	childIDs, exists := a.f.Instance.SpendChildren(a.f.SpendID(spendAlias))
+	require.True(a.f.test, exists, "Conflict %s does not exist", spendAlias)
+
+	require.Equal(a.f.test, len(childAliases), childIDs.Size(), "Conflict %s has wrong number of children", spendAlias)
+	for _, childAlias := range childAliases {
+		require.True(a.f.test, childIDs.Has(a.f.SpendID(childAlias)), "Conflict %s does not have child %s", spendAlias, childAlias)
+	}
+}
+
+// Parents asserts that the given spend has the given parents.
+func (a *Assertions) Parents(spendAlias string, parentAliases ...string) {
+	parents, exists := a.f.Instance.SpendParents(a.f.SpendID(spendAlias))
+	require.True(a.f.test, exists, "Conflict %s does not exist", spendAlias)
+
+	require.Equal(a.f.test, len(parentAliases), parents.Size(), "Conflict %s has wrong number of parents", spendAlias)
+	for _, parentAlias := range parentAliases {
+		require.True(a.f.test, parents.Has(a.f.SpendID(parentAlias)), "Conflict %s does not have parent %s", spendAlias, parentAlias)
+	}
+}
+
+// LikedInstead asserts that the given spends return the given LikedInstead spends.
+func (a *Assertions) LikedInstead(spendAliases []string, likedInsteadAliases ...string) {
+	likedInsteadConflicts := a.f.LikedInstead(spendAliases...)
+
+	require.Equal(a.f.test, len(likedInsteadAliases), likedInsteadConflicts.Size(), "LikedInstead returns wrong number of spends %d instead of %d", likedInsteadConflicts.Size(), len(likedInsteadAliases))
```

Comment:
```suggestion
	likedInsteadSpends := a.f.LikedInstead(spendAliases...)

	require.Equal(a.f.test, len(likedInsteadAliases), likedInsteadSpends.Size(), "LikedInstead returns wrong number of spends %d instead of %d", likedInsteadSpends.Size(), len(likedInsteadAliases))
```